scripts/build-example.sh

```diff
@@ -4,10 +4,13 @@ set -eo pipefail
 
 dir=${dir-"examples/compiled"}
 
+# mac osx doesn't have ms time, need to use python -- https://apple.stackexchange.com/a/314573
+start=`python -c 'from time import time; print int(round(time() * 1000))'`
```

Comment:
Python 2 is dead. Let's use Python 3's print function.

Reply:
Actually, perl is ways faster `perl -MTime::HiRes=time -e 'printf "%d\n", time*1000'`.